src/test/kotlin/com/autonomousapps/internal/ManifestParserTest.kt

```diff
@@ -150,26 +152,60 @@ class ManifestParserTest {
     assertThat(manifest.applicationName).isEqualTo("mutual.aid.explode")
   }
 
-  @Test fun `parse themes`() {
+  @Test fun `parse resource references`() {
```

Comment:
In this test, I added common references that we are using in our own app. This is definitely not an exhaustive list of all places where references are allowed.